### ISSUE TITLE
Can not deploy assets to aws with serverless-lift plugin using V4

### ISSUE THREAD

**Issue by MrMySQL** [serverless/serverless#12797]
### Issue description

when using server less v4 (any version) when trying to upload some assets to aws with a pretty default template, I get an error: `Cannot destructure property 'name' of 'undefined' as it is undefined at _Progress.create (file:///Users/mrmysql/.serverless/releases/4.1.23/package/dist/sf-core.js:76:1138)`

While with serverless v3 everything is ok.

```
constructs:
    website:
        type: server-side-website
        assets:
            '/css/*': public/css
            '/js/*': public/js
```
public/css and  public/js contain files.

Here is the full output:
```
✗ serverless website:assets:upload

(node:45842) [DEP0040] DeprecationWarning: The `punycode` module is deprecated. Please use a userland alternative instead.
(Use `node --trace-deprecation ...` to show where the warning was created)
[WARNING] aws-cdk-lib.aws_cloudfront_origins.S3Origin is deprecated.
  Use `S3BucketOrigin` or `S3StaticWebsiteOrigin` instead.
  This API will be removed in the next major release.
[WARNING] aws-cdk-lib.aws_cloudfront_origins.S3Origin is deprecated.
  Use `S3BucketOrigin` or `S3StaticWebsiteOrigin` instead.
  This API will be removed in the next major release.
[WARNING] aws-cdk-lib.aws_cloudfront_origins.S3Origin#bind is deprecated.
  Use `S3BucketOrigin` or `S3StaticWebsiteOrigin` instead.
  This API will be removed in the next major release.
[WARNING] aws-cdk-lib.aws_cloudfront_origins.S3Origin#bind is deprecated.
  Use `S3BucketOrigin` or `S3StaticWebsiteOrigin` instead.
  This API will be removed in the next major release.
Deploying the assets for the 'website' website


✖ Cannot destructure property 'name' of 'undefined' as it is undefined.
TypeError: Cannot destructure property 'name' of 'undefined' as it is undefined.
    at _Progress.create (file:///Users/mrmysql/.serverless/releases/4.3.2/package/dist/sf-core.js:76:1138)
    at _ServerSideWebsite.uploadAssets (/Users/mrmysql/projects/test/node_modules/serverless-lift/dist/src/constructs/aws/ServerSideWebsite.js:209:33)
    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)
    at async _ServerSideWebsite.uploadAssetsCommand (/Users/mrmysql/projects/test/node_modules/serverless-lift/dist/src/constructs/aws/ServerSideWebsite.js:191:5)
    at async PluginManager.runHooks (file:///Users/mrmysql/.serverless/releases/4.3.2/package/dist/sf-core.js:913:9309)
    at async PluginManager.invoke (file:///Users/mrmysql/.serverless/releases/4.3.2/package/dist/sf-core.js:913:10080)
    at async PluginManager.run (file:///Users/mrmysql/.serverless/releases/4.3.2/package/dist/sf-core.js:913:10811)
    at async Serverless.run (file:///Users/mrmysql/.serverless/releases/4.3.2/package/dist/sf-core.js:920:10612)
    at async runFramework (file:///Users/mrmysql/.serverless/releases/4.3.2/package/dist/sf-core.js:943:2670)
    at async route (file:///Users/mrmysql/.serverless/releases/4.3.2/package/dist/sf-core.js:966:29270)

For help, try the following:
  • Run the command again with the "--debug" option
  • Run "serverless support"
  • Review the docs: https://www.serverless.com/framework/docs/
```

### Context

✗ serverless support --summary

(node:47852) [DEP0040] DeprecationWarning: The `punycode` module is deprecated. Please use a userland alternative instead.
(Use `node --trace-deprecation ...` to show where the warning was created)
Serverless ϟ Support

This generates a report from your last Serverless Framework command (including any errors) to use for Github Issues, debugging w/ AI, or creating a support ticket w/ Serverless Inc.

Summary Report -----------------

# Service Overview
- Serverless Framework Version: 4.3.2
- Service Path: /Users/mrmysql/projects/test
- Service Config File: serverless.yml
- Service Name: symfony
- Service Runtime: provided.al2
- Service Stage: dev
- Service Region: us-east-1
- Command: website:assets:upload 

## Error Message
Cannot destructure property 'name' of 'undefined' as it is undefined.

## Error Stacktrace
```
at _Progress.create (file:///Users/mrmysql/.serverless/releases/4.3.2/package/dist/sf-core.js:76:1138)
at _ServerSideWebsite.uploadAssets (/Users/mrmysql/projects/test/node_modules/serverless-lift/dist/src/constructs/aws/ServerSideWebsite.js:209:33)
at process.processTicksAndRejections (node:internal/process/task_queues:105:5)
at async _ServerSideWebsite.uploadAssetsCommand (/Users/mrmysql/projects/test/node_modules/serverless-lift/dist/src/constructs/aws/ServerSideWebsite.js:191:5)
at async PluginManager.runHooks (file:///Users/mrmysql/.serverless/releases/4.3.2/package/dist/sf-core.js:913:9309)
at async PluginManager.invoke (file:///Users/mrmysql/.serverless/releases/4.3.2/package/dist/sf-core.js:913:10080)
at async PluginManager.run (file:///Users/mrmysql/.serverless/releases/4.3.2/package/dist/sf-core.js:913:10811)
at async Serverless.run (file:///Users/mrmysql/.serverless/releases/4.3.2/package/dist/sf-core.js:920:10612)
at async runFramework (file:///Users/mrmysql/.serverless/releases/4.3.2/package/dist/sf-core.js:943:2670)
at async route (file:///Users/mrmysql/.serverless/releases/4.3.2/package/dist/sf-core.js:966:29270)
```

## Service Config
```
# Read the documentation at https://www.serverless.com/framework/docs/providers/aws/guide/serverless.yml/
service: symfony
provider:
    name: aws
    # The AWS region in which to deploy (us-east-1 is the default)
    region: us-east-1
    # The stage of the application, e.g. dev, production, staging… ('dev' is the default)
    stage: dev
    runtime: provided.al2
    environment:
        # Symfony environment variables
        APP_ENV: dev
plugins:
  - ./vendor/bref/bref
  - serverless-lift
functions:
    # This function runs the Symfony website/API
    web:
        handler: public/index.php
        timeout: 28 # in seconds (API Gateway has a timeout of 29 seconds)
        layers:
            - ${bref:layer.php-82-fpm}
        events:
            - httpApi: '*'
    # This function let us run console commands in Lambda
    # console:
    #     handler: bin/console
    #     timeout: 120 # in seconds
    #     layers:
    #         - ${bref:layer.php-82} # PHP
    #         - ${bref:layer.console} # The "console" layer
package:
    patterns:
        # Excluded files and folders for deployment
        # - '!assets/**'
        - '!node_modules/**'
        # - '!public/build/**'
        # - '!tests/**'
        - '!var/**'
        # If you want to include files and folders that are part of excluded folders,
        # add them at the end
        - 'var/cache/prod/**'
        # - 'public/build/entrypoints.json'
        # - 'public/build/manifest.json'
constructs:
    website:
        type: server-side-website
        assets:
            '/css/*': public/css
            '/js/*': public/js
            # add here any file or directory that needs to be served from S3
```

# Service Overview
- Serverless Framework Version: 4.3.2
- Command: support --summary true

**Comment by czubocha** [comment]
Hey @MrMySQL @nolotz, this has been addressed in version 4.4.4, which is now live. Feel free to try it out and let us know if you run into any issues.

**Comment by czubocha** [comment]
Going forward, it should be used with arguments, at least with `message`.
